### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, 1:30 pm, Friday Apr 1, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2957] [Meeting Agendas]
The agenda for this coming week is as follows:

1. Dong update
2. Peter update
3. Jason update
4. Cold cases
a. @balacij #849
b. @cd155 #473
c. @peter-michalski #759
d. @smiths #2955
5. Questions/Comments/Concerns/Discussion

EDIT: [Meeting Record (for McMaster students/faculty only)](https://mcmasteru365.sharepoint.com/sites/SummerDrasil/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FSummerDrasil%2FShared%20Documents%2FGeneral%2FRecordings%2FDrasil%20Meeting%2D20220401%5F133420%2DMeeting%20Recording%2Emp4&parent=%2Fsites%2FSummerDrasil%2FShared%20Documents%2FGeneral%2FRecordings)

**Comment by smiths** [comment]
@cd155 and @peter-michalski, please add new "cold issues" when you get a chance.

**Reply by cd155** [reply]
A cold case ticket added

**Comment by JacquesCarette** [comment]
So one thing I've noticed is that discussions can't be "closed". So all the agendas end up being listed in the discussions page, which I don't think is good.

Maybe we could use part of the Wiki instead? We could set aside a part of the wiki for 'temporal' items. It's nice to have a history of this stuff kept. We could use issues too, and then close them.  It is possible to 'delete' discussions, but I don't want to do that.

**Comment by smiths** [comment]
I agree that it would be nice if we could "close" discussions.  If the Discussions page let us create a separate forum for agendas, that might work to give us a place to archive them, but I don't see an option for that.

What about if we use the issue tracker for our agendas?  We could create a label for meeting agenda so that we could mark all meetings.  We can then close the issue after the meeting.  We could try the wiki idea, but it feels less integrated into our regular work flow.

**Comment by cd155** [comment]
There are categories in the discussion. If you open the discussion, it will show on the left side. The default view is view all, so all the discussions show up. There is an option to add the customized categories, it just works like a label in the issue tracker. I just create a category called Test(Close). 

I believe current all discussions are in the General category, if we change the category to Close once the agenda is over, the general category will not have this agenda.

**Comment by smiths** [comment]
Thank you for the information @cd155.  I looked for creating custom categories, but I couldn't find it.  :smile:  I found it now and relabelled your category to be "Meeting Agendas".  They won't appear under the list of Discussion threads, unless someone selects this category, or All.  

If we want something to be the equivalent of closing an issue, we could categorize the agendas as General Discussion and then move them to the category of Meeting Archives after the meeting.  We would then only have the most recent meeting under General Discussion.

**Comment by JacquesCarette** [comment]
Coming back to this: I think maybe moving these to issues would still be better.

**Comment by smiths** [comment]
Sure we can do that going forward.
